html/includes/common/availability-map.inc.php

```diff
@@ -226,16 +226,17 @@
                 $deviceLabelOld = 'availability-map-oldview-box-down';
                 $host_down_count++;
             }
+            $deviceSystemName = ip_to_sysname($device, $device['hostname']);
```

Comment:
Thanks for this, would you mind changing this variable to lower case please, you can use `_` to show a separation pls.